graph/vertex/impl/AttributeVertexImpl.java

```diff
@@ -74,7 +76,12 @@
 
         @Override
         public Encoding.Status status() {
-            return Encoding.Status.IMMUTABLE;
+            return isPersisted() ? PERSISTED : BUFFERED;
+        }
```

Comment:
interesting new change: because we do a true PUT on attributes, we actually always know whether an attribute is persisted or buffered - this aligns them with all other concept instances, which are also either buffered or persisted!